### DISCUSSION TITLE
Helix runtime inaccessible

### DISCUSSION THREAD

**Discussion by jwielink** [helix-editor/helix#6445] [General]
### Summary

After install `choco install helix` the runtime directory is inaccessible even though the %HELIX_RUNTIME% environment variable is set. `hx --health` will return that the runtime directory does not exist. Also tutor and themes are not working (might be related to https://github.com/helix-editor/helix/issues/5050).

### Reproduction Steps

 

I tried this:
```
cmd> echo %HELIX_RUNTIME%
%ChocolateyInstall%\lib\helix\tools\helix-22.08-x86_64-windows\runtime

cmd> hx --health
Config file: default
Language file: default
Log file: C:\Users\jordy\AppData/Local/helix\helix.log
Runtime directory: %ChocolateyInstall%\lib\helix\tools\helix-22.08-x86_64-windows\runtime
Runtime directory does not exist.
Clipboard provider: clipboard-win
System clipboard provider: clipboard-win
```
not working, although the runtime directory is set to the correct location, my ChocolateyInstall variable is also set:
```
cmd> echo %ChocolateyInstall%
C:\ProgramData\chocolatey
```
Restarting terminal after install or restart of pc did not resolve the issue.
Next I ran choco's RefreshEnv.cmd:
```
cmd> RefreshEnv.cmd
Refreshing environment variables from registry for cmd.exe. Please wait...Finished..
```
now the issue is resolved:
```
cmd> hx --health
Config file: default
Language file: default
Log file: C:\Users\jordy\AppData/Local/helix\helix.log
Runtime directory: C:\ProgramData\chocolatey\lib\helix\tools\helix-22.08-x86_64-windows\runtime
Clipboard provider: clipboard-win
System clipboard provider: clipboard-win
```

### Platform

Windows

### Terminal Emulator

wt

### Helix Version

helix 22.08 (ee94031f)

**Comment by prulde** [comment]
When running wt as admin:
```
>$env:HELIX_RUNTIME
C:\ProgramData\chocolatey\lib\helix\tools\helix-22.08-x86_64-windows\runtime
```
Without:
```
>$env:HELIX_RUNTIME
%ChocolateyInstall%\lib\helix\tools\helix-22.08-x86_64-windows\runtime
```
You can overridde `HELIX_RUNTIME` enviroment variable in powershell profile, but first you need to change the default powershell execution policy to enable profiles:
```
>Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
```
Then you can run wt without admin rights:
```
>notepad $profile
```
```
$env:HELIX_RUNTIME="C:\ProgramData\chocolatey\lib\helix\tools\helix-22.08-x86_64-windows\runtime"
```
